operations/tasks/tag-buildroot.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Tag the desired build image with the post build tag (“staged” for stage, “released” for prod)
+- name: tag buildroot imagestream
+  command: >
+    oc tag {{ osbs_buildroot_imagestream }}:{{ osbs_buildroot_live_tag }} {{ osbs_buildroot_imagestream }}:{{ osbs_buildroot_imagestream_tag }}
```

Comment:
Are these the right way round? The examples in the help output suggest they are not:
```
Examples:
  # Tag the current image for the image stream 'openshift/ruby' and tag '2.0'
into the image stream 'yourproject/ruby with tag 'tip'.
  oc tag openshift/ruby:2.0 yourproject/ruby:tip
```

Do we need `--source=imagestreamtag` (or whatever), or will it work that out itself?

And later, we import the image tag from the repository -- does that mean the thing we're trying to create a tag for doesn't exist yet? Is that ok?

Reply:
live_tag is 1.0-78 and osbs_buildroot_imagestream_tag is staged/released

Reply:
That wasn't clear to me. Maybe `osbs_buildroot_imagestream_tag` should be named `osbs_buildroot_imagestream_post_build_tag` (or just `imagstream_post_build_tag`) to make that clearer?

Reply:
sure

Reply:
# Do we need --source=imagestreamtag (or whatever), or will it work that out itself?
it will work out itself

# And later, we import the image tag from the repository -- does that mean the thing we're trying to 
# create a tag for doesn't exist yet? Is that ok?
it will have return code 1 and task will fail